src/jenkins.py

```diff
@@ -308,18 +310,37 @@ def _unlock_wizard(container: ops.Container) -> None:
     )
 
 
-def _install_configs(container: ops.Container) -> None:
+def _install_jenkins_config(container: ops.Container, filename: str) -> None:
     """Install jenkins-config.xml.
 
     Args:
         container: The Jenkins workload container.
+        filename: the source file to copy contents from.
     """
-    with open("templates/jenkins-config.xml", encoding="utf-8") as jenkins_config_file:
+    with open(filename, encoding="utf-8") as jenkins_config_file:
         container.push(CONFIG_FILE_PATH, jenkins_config_file, user=USER, group=GROUP)
```

Comment:
Shouldn't ops.pebble.PathError be handled here?

Reply:
Updated